code/modules/organs/external/_external.dm

```diff
@@ -332,7 +334,9 @@
 	if(status & ORGAN_SPLINTED)
 		. += 0.5
 
-	. += (-(limb_efficiency / 100 - 1) * 3)	//0 at 100 efficiency, -1.5 at 150, +1.5 at 50
+	var/muscle_eff = owner.get_specific_organ_efficiency(OP_MUSCLE, organ_tag)
+	muscle_eff = muscle_eff - (muscle_eff/(owner.get_specific_organ_efficiency(OP_NERVE, organ_tag)/100)) //Need more nerves to control those new muscles
+	. += max(-(muscle_eff/ 100 - 1)/4, MAX_MUSCLE_SPEED)
```

Comment:
The `-1` in this calculation causes slowdown of .25 per leg for every person, due to the base from the previous line being 0 instead of the previous 1. Removing the `-1` should fix it.